packages/components/src/settings/feature-options-defaults.json

```diff
@@ -197,6 +197,26 @@
       "minValue": 1,
       "maxValue": 5000,
       "values": []
+    },
+    "enableMouseMoveLimitedByIdCheck": {
+      "defaultValue": false,
+      "labelText": "Enable Mouse Move on Google Docs Element Id:",
+      "boolean": true,
+      "values": [
+        {
+          "value": true,
+          "text": "Yes"
+        },
+        {
+          "value": false,
+          "text": "No"
+        }
+      ]
+    },
+    "mouseMoveLimitedById": {
+      "defaultValue": "docs-editor-container",
+      "labelText": "Limited by the id",
+      "text": true
```

Comment:
so, per my comment in the conversation, I was thinking we could have just one setting, e.g.

```
  "googleDocsMouseMoveLimitedById": {
      "defaultValue": "docs-editor-container",
      "labelText": "Enable Mouse Move on Google Docs Element Id",
      "text": true
}
```

And to fine tune it we could use logic like:

If this wasn't set, then the auto-enable on google docs could be off, 
if it is set to "*" it would mean enable on all elements on google docs
If it is set to any other string it would mean enable on elements with that id on google docs

What do you think?

Reply:
From my point of view - it is redundantly, because:

1. lokup click event doesn't work properly on google docs, so if we won't enable it by default (as suggested in the issue) - it could confuse users. 
2. it is not clear for most person to use * or `docs-editor-container` to get it work.

I believe that we think about different types of users :)
I thought that we want to help those persons who :
   - won't guess to enable mouseMove manually in settings
   - would confuse when google docs menu would close popup with ready words and would execute lookup on menu titles

I suggest even turn on `enableMouseMoveLimitedByIdCheck` by default.
This way we would protect less It-known persons and give other the ability to tune it (that's why I suggested before to print setting for id name). 

Of course if you think that your suggestion is better - I would implement it :)

Reply:
ok, we can keep the two settings if you think it's important. I think we should make these further changes then:

1) mouseMoveLimitedById unprivate - i.e. so that users can change it if they need to (e.g. if this changes on google docs before we can get a new release with a new default value out)

2) change the labelText for enableMouseMoveLimitedByIdCheck to "Auto-enable Mouse Move on Google Docs"

3) change the labelText for mouseMoveLimitedById to "Google Docs Container Id"

I also still think that it would be better if both of these setting names explicitly referred to google docs. e.g. I would use the following names: enableMouseMoveOnGoogleDocs and googleDocsContainerId

Reply:
>change the labelText for enableMouseMoveLimitedByIdCheck to "Auto-enable Mouse Move on Google Docs"

Bridget, I didn't finally understand - does it mean that mouseMove shouldn't be enabled without this check?
and there is no check that defines that we limit by any selector? we should limit, if the mouseMoveLimitedById has value?

Reply:
>Bridget, I didn't finally understand - does it mean that mouseMove shouldn't be enabled without this check?

Ok, sorry, I see that I am confusing things. The code currently ALWAYS auto-enables mousemove on google docs and this setting currently controls only if the mousemove will be limited to id, right?

If so, then let's keep that as it is, and change the labelText to 

"Limit Mouse Move on Google Docs to top-level container"

I think we need some usability testing on this generally, so I will stop sending us in circles until we have that. Sorry!

Reply:
> The code currently ALWAYS auto-enables mousemove on google docs and this setting currently controls only if the mousemove will be limited to id, right?

Yes, it works this way:

```
enableMouseMoveEvent () {
    return this.platform.isDesktop && 
              (this.featureOptions.items.enableMouseMove.currentValue || 
               this.options.enableMouseMoveOverride || this.platform.isGoogleDocs)
  }
```

We enable mouseMove in the following case (only for Desktop):
   - enableMouseMove is checked in options (was before)
   - we are on googleDocs

when MouseMove is registered for the listener, it defines limitedBy this way:
```
this.limitedById = Platform.getIsGoogleDocs() && 
                             eventParams.enableMouseMoveLimitedByIdCheck ? 
                             eventParams.mouseMoveLimitedById : null
```

so limited by Id check would work only when it is googleDocs and limitedByIdCheck 

So that's why - @balmas, you understood right!